vendor/github.com/asaskevich/govalidator/validator.go

```diff
@@ -323,12 +323,12 @@ func IsNull(str string) bool {
 
 // HasWhitespaceOnly checks the string only contains whitespace
 func HasWhitespaceOnly(str string) bool {
-    return len(str) > 0 && rxHasWhitespaceOnly.MatchString(str)
```

Comment:
Why are you changing a single file in the vendor folder?

Reply:
I was inspecting these sources, and I think my editor must have run `gofmt` and saved.

Reply:
Re-vendored and force pushed.